app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
+import { IGitTrigger } from 'core/domain';
+import { SETTINGS } from 'core/config';
+
+export function ImportDeliveryConfigExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+  const trigger = props.execution.trigger as IGitTrigger;
+  const errorDetailsAvailable = stage.isFailed && !stage.failureMessage && stage.context.error;
+  const manifestDirectory = SETTINGS.managedDelivery?.manifestBasePath + (stage.context.directory ?? '');
+  const manifestFilename = stage.context.manifest ?? SETTINGS.managedDelivery?.defaultManifest;
+  const gitRef = stage.context.ref ?? 'refs/heads/master';
+
+  return (
+    <ExecutionDetailsSection name={props.name} current={props.current}>
+      <div className="row">
+        <div className="col-md-12">
+          <h5>Import Delivery Config Stage Configuration</h5>
```

Comment:
I suggest nuking this h5 entirely, it doesn't seem like it's conveying anything useful.